locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -365,6 +365,16 @@ public boolean isWindows(NodeMetadata node, ConfigBag config) {
                 : (OsFamily.WINDOWS == confFamily);
     }
 
+    public boolean isLocationFirewalldEnabled(SshMachineLocation location) {
+        int result = location.execCommands("checking if firewalld is active", 
+                ImmutableList.of(IptablesCommands.firewalldServiceIsActive()));
+        if (result == 0) {
+            return true;
+        }
+        
```

Comment:
@neykov here is addressed your previous comment regarding the firewall-cmd existence